public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -106,6 +107,7 @@ export function AppChrome({ children }: Props) {
           {menuDockedAndOpen && (
             <MegaMenu className={styles.dockedMegaMenu} onClose={() => chrome.setMegaMenuOpen(false)} />
           )}
+          {!state.chromeless && <ScopesDashboards />}
```

Comment:
Ah, this is another case we need to cover when designing the global scope selector, cc @Ijin08